clients/replication/replication.go

```diff
@@ -17,12 +17,14 @@ type Client struct {
 
 type CreateParams struct {
 	clients.OrgParams
-	Name           string
-	Description    string
-	RemoteID       string
-	LocalBucketID  string
-	RemoteBucketID string
-	MaxQueueSize   int64
+	Name                   string
+	Description            string
+	RemoteID               string
+	LocalBucketID          string
+	RemoteBucketID         string
+	MaxQueueSize           int64
+	DropNonRetryableData   bool
+	NoDropNonRetryableData bool
```

Comment:
You could probably get away with only 1 flag here (for whatever isn't the default in the API), but not a blocker if you want to keep 2 flags for symmetry with `update`.